homeassistant/components/light/rflink.py

```diff
@@ -185,6 +185,18 @@ def brightness(self):
         """Return the brightness of this light between 0..255."""
         return self._brightness
 
+    @property
+    def device_state_attributes(self):
+        """Return the device state attributes."""
+        attr = {}
+        super_attr = super().device_state_attributes
```

Comment:
I can't find any parent class with this implemented besides the base class, which is just a template.

Reply:
Done.